lienzo-core/src/main/java/com/ait/lienzo/client/core/shape/AbstractMultiPointShape.java

```diff
@@ -175,10 +175,10 @@ private DefaultMultiPointShapeHandleFactory(final AbstractMultiPointShape<?> sha
         @Override
         public Map<ControlHandleType, IControlHandleList> getControlHandles(final List<ControlHandleType> types) {
             if ((null == types) || (types.isEmpty())) {
-                return null;
+                return new HashMap<>();
```

Comment:
Hi @jomarko,

did you check all `getControlHandles` method invocations? If I correct some of them already relies on `null`s so this small change can lead to further issues.

Reply:
same here